guide/getting-started/visual-language.md

```diff
@@ -99,9 +98,14 @@ On February 24, 2010, Satoshi posted an updated icon that adopted the capital 
    layout = "float-left"
 %}
 
+</div>
```

Comment:
Did you intentionally remove the 2-column layout here? If so, seems inconsistent with the paragraph above. If not, just move the closing div a few lines lower, below the text.

Reply:
Fixed